README.md

```diff
@@ -0,0 +1,6 @@
+# Captcha.EU 
+Typo3 extension for captcha.eu see <a href="https://docs.captcha.eu/typo3-install?id=-form-beta">Documentation</a>
+
+
+# Credits
+this extension is heavily inspired by the great work from the folks at [Studio Mitte](https://studiomitte.com) checkout out there other extensions [here](https://www.studiomitte.com/loesungen/typo3)

```

Comment:
Looks good, thanks! however in terms of grammar, I would suggest to start with `This`